scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -261,8 +262,20 @@ class FossId internal constructor(
             filteredPackages.forEach { pkg ->
                 val startTime = Instant.now()
 
-                // TODO: Continue the processing of other packages and add an issue to the scan result.
-                require(pkg.vcsProcessed.type == VcsType.GIT) { "FossID only supports Git repositories." }
+                if (pkg.vcsProcessed.type != VcsType.GIT) {
```

Comment:
+1 for the implementation. It would be great if a test case could be added for this scenario.

Reply:
I've added a test. Please double-check that I'm not doing something stupid / superfluous there.

Reply:
Thanks for adding the test. It looks already good. You could even improve it by passing a second package to the scanner and verify that it gets processed after the error with the unsupported VCS is handled. But I leave this to you.